### ISSUE TITLE
Adding Yaw and UUID from a DroneID broadcast

### ISSUE THREAD

**Issue by Tissy** [alphafox02/antsdr_dji_droneid#4]
Would I be right in thinking that the DJI firmware is not-currently open source?

There are a couple of elements from a DJI broadcast which I would like to add, such as aircraft yaw and UUID.

Any suggestion how these could be implemented please, happy to assist if needed.

**Comment by alphafox02** [comment]
I don’t believe so, but I could try and ask. Maybe it is something they’d be able to add if it’s not open source.

**Comment by Tissy** [comment]
That'll be excellent, thank you.  I have the information you may need from an Aeroscope if required.

**Comment by lone-boy** [comment]
@Tissy Hello, due to the encryption mechanism of the droneID project, this project has been closed. I don't quite understand what this UUID refers to. Is it the serial number of the aircraft? If you have more information about droneID, you can contact this email loneboy@microphase.cn.

**Comment by Tissy** [comment]
Thank you @lone-boy , email sent.  Look forward to hearing from you.

**Comment by waseem-arshad94** [comment]
Dear @Tissy;

Please Can you tell me about the models on which DJI make the ID encrypted?

Thanks
